puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/rubygemRpmNightlyRelease.groovy

```diff
@@ -0,0 +1,31 @@
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage ("Build Gem") {
+            steps {
+                dir('gem-build') {
+                    git(url: "https://github.com/${env.OWNER_REPO}", branch: env.BRANCH)
+                    sh("gem build ${env.GEMSPEC}")
+                    archiveArtifacts(artifacts: '*.gem')
+                }
+            }
+        }
+
+        stage('Build RPM') {
+            steps {
+                dir('foreman-packaging') {
+                    git(url: 'https://github.com/theforeman/foreman-packaging', branch: 'rpm/develop')
+                    obal(action: "release", extraVars: ['build_package_tito_releaser_args': ["--arg jenkins_job=${env.JOB_NAME}", "--arg jenkins_job_id=${env.BUILD_ID}"]], packages: env.PACKAGE_NAME)
```

Comment:
Can we also set the releaser via variables here? It's what the current RPM build job does and I think that unless we have a specific `obal` command it's the quickest way to add more nightlies.

Reply:
It looks like obal may have an undocumented `build_package_tito_releasers` variable we can set. I will play with that a bit and get back to you.

Reply:
my bad, looks like it's just `releasers`.